certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Kyndryl mainframe connector",
+    "version": "1.0.0",
+    "title": "kyndryl"
+  },
+  "host": "kyndryl-apigateway.azure-api.net",
+  "basePath": "/",
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/cb12-policy/getpolicy": {
+      "get": {
+        "tags": [
+          "cb12-policy"
+        ],
+        "operationId": "getPolicy",
+        "parameters": [
+          {
+            "name": "CUSTOMERNUMBER",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/getPolicy_response_200"
+            }
+          }
+        },
+        "summary": "Get policy"
```

Comment:
@RTkyndryl The 'description' property is required. please add description under the summary. 
```
"summary": "Get policy",
"description": "Your description"
```